### PR TITLE
Adding a confidential client {backend, frontend}

### PR DESCRIPTION
**Pull request by pbugni**

### CODE REVIEW

File: dev/docker-compose.yaml [hunk 1]
```diff
@@ -102,6 +154,8 @@ services:
       # NB accessControlAllowOrigin is deprecated, but not noted in docs
       # https://github.com/traefik/traefik/issues/8796
       - "traefik.http.middlewares.fhirwall-${COMPOSE_PROJECT_NAME}-cors.headers.accessControlAllowOriginList=*"
+      # allow all verbs used by FHIR REST
+      - "traefik.http.middlewares.fhirwall-${COMPOSE_PROJECT_NAME}-cors.headers.accessControlAllowMethods=HEAD,GET,OPTIONS,PATCH,POST,PUT,DELETE"
```
Comment (ivan-c) [inline dev/docker-compose.yaml:158]: I think these CORS header changes are only necessary if we expect the frontend to talk directly to `fhirwall` (ala ISACC) vs through the sof-backend fhir-router.

**Review by ivan-c — APPROVED** [review]
Looks good! Ready to merge, but may have found some extra header changes

File: dev/docker-compose.yaml (hunk 1 above)
Comment (pbugni) [reply dev/docker-compose.yaml:158]: wow, great catch.  thank you.

Comment (pbugni) [reply dev/docker-compose.yaml:158]: on second thought, we have both confidential AND public clients included in this demo.  let's keep this for now as it is required by the public ones.

**Pull request merged**